integrations/symfony-full-configuration.rst

```diff
@@ -78,6 +78,7 @@ This page shows an example of all configuration values provided by the bundle.
                 service: 'my_service'       # Can not be used with "factory" or "config"
                 flexible_client: false      # Can only be true if http_methods_client is false
                 http_methods_client: false  # Can only be true if flexible_client is false
+                public: false               # Can be true to make the service public
```

Comment:
Lets be a bit more explicit: "Set true if you really can't use dependency injection and need to make the client service a public service"